adv_project/settings.py

```diff
@@ -96,6 +96,9 @@
     }
 }
 
+db_from_env = dj_database_url.config(conn_max_age=500)
+DATABASES['default'].update(db_from_env)
```

Comment:
This is interesting syntax. It may be the first time I've seen it so far but it's very semantic which is good!